recipes/elemental-core/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "elemental-core" %}
+{% set version = "0.2.1.dev0" %}
+{% set md5 = "3244b9a1d62b4e180c37cea88db5931c" %}
```

Comment:
The PyPI and sha256 info has been added to the docs.

artPlusPlus/conda-forge.github.io#348